nni/algorithms/compression/v2/pytorch/pruning/tools/task_generator.py

```diff
@@ -227,7 +227,10 @@ def _init_config_sparsity(self, config: Dict) -> Tuple[List[Dict], List]:
         op_names = config['op_names']
 
         if target_sparsity == 0:
```

Comment:
in which case `target_sparsity` is 0?

Reply:
as in description example, if `origin_masks` already masks 50%, and sparsity set in `config_list` is 50%, then the target_sparsity is 0.